datumaro/tests/test_coco_format.py

```diff
@@ -39,18 +38,20 @@ def __iter__(self):
                                 ), label=0,
                                 id=2, group=2, attributes={'is_crowd': True}),
                         ]
-                    ),
-                ])
+                    )]
 
-            def categories(self):
-                label_cat = LabelCategories()
-                label_cat.add('TEST')
-                return { AnnotationType.label: label_cat }
+        # Creating categories
+        label_cat = LabelCategories()
+        label_cat.add('TEST')
+        categories = { AnnotationType.label : label_cat}
+
+        # Creating exemplary dataset
+        exemplary_dataset = Dataset.from_iterable(items_list, categories)
```

Comment:
Consider creating and passing the first argument in-place to further simplify code. There is no need in a separate variable for the list above.